### PR TITLE
PLANNER-2495 Quickstarts distribution

### PR DESCRIPTION
**Pull request by rsynek**
### JIRA
https://issues.redhat.com/browse/PLANNER-2495

### Referenced pull requests
https://github.com/kiegroup/optaplanner/pull/1509

### CODE REVIEW

**Review by Ginxo — APPROVED** [review]
LGTM

File: build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Welcome to OptaPlanner
```
Comment (dupliaka) [inline build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt:1]: Shell we add any pre-requirements like JDK version/heap size here?

Comment (rsynek) [reply build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt:1]: Not sure we need to go into this detail; the `runExamples` and `runQuickstarts` scripts take care both of the Java version and memory settings.

File: build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+======================
+
+Run the quickstarts
+-------------------
+
+On Linux, Mac or Cygwin:
+  runQuickstarts.sh
```
Comment (yurloc) [inline build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt]:
```suggestion
  ./runQuickstarts.sh
```

File: build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+----------------------------
+
+Please refer to https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/README.adoc.
+
+Run the examples
+----------------
+
+It's easy to run the examples:
+On Linux, Mac or Cygwin:
+  runExamples.sh
```
Comment (yurloc) [inline build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt]:
```suggestion
  ./runExamples.sh
```

File: build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+Run the quickstarts in IDE
+----------------------------
+
+Please refer to https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/README.adoc.
+
+Run the examples
+----------------
+
+It's easy to run the examples:
+On Linux, Mac or Cygwin:
```
Comment (yurloc) [inline build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt]:
```suggestion
On Linux, macOS or Cygwin:
```

File: build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt [hunk 5]
```diff
@@ -0,0 +1,74 @@
+Welcome to OptaPlanner
+======================
+
+Run the quickstarts
+-------------------
+
+On Linux, Mac or Cygwin:
```
Comment (yurloc) [inline build/optaplanner-distribution/src/main/assembly/resources/ReadMe.txt]:
```suggestion
On Linux, macOS or Cygwin:
```

File: build/optaplanner-distribution/src/main/assembly/assembly-optaplanner-quickstarts.xml [hunk 6]
```diff
@@ -43,46 +48,63 @@
...
+      </includes>
+      <useProjectArtifact>false</useProjectArtifact>
+      <outputDirectory/>
+      <useStrictFiltering>true</useStrictFiltering>
+      <unpack>true</unpack>
+      <!--<unpackOptions>
+        <includes>
+          <include>examples/**</include>
+        </includes>
+      </unpackOptions>-->
```
Comment (yurloc) [inline build/optaplanner-distribution/src/main/assembly/assembly-optaplanner-quickstarts.xml]: Commented out code. Does it have a purpose?

**Review by triceo — APPROVED** [review]

File: build/optaplanner-distribution/src/main/assembly/assembly-optaplanner-quickstarts.xml (hunk 6 above)
Comment (rsynek) [reply build/optaplanner-distribution/src/main/assembly/assembly-optaplanner-quickstarts.xml]: good catch, let me remove it

**Review by almope — APPROVED** [review]

**Pull request merged**